cogboard-webapp/src/components/widgets/types/ToDoListWidget.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { array } from 'prop-types';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import { Typography } from '@material-ui/core';
+import { postWidgetContentUpdate } from '../../../utils/fetch';
+import { shallowEqual, useSelector } from 'react-redux';
+import styled from '@emotion/styled/macro';
+
+const StyledCheckbox = styled(Checkbox)`
+  margin-left: 15px;
+
+  &.Mui-checked {
+    color: rgba(255, 255, 255, 0.7);
+  }
+`;

```

Comment:
Please use the other convention when creating styled widget:
Create a folder `ToDoListWidget` with:

- index.js - widget impl
- style.js - all the styling

see `CheckboxWidget` as an example

Reply:
Yeah,  I tried that before, but it didn't work for me. Maybe we can catchup?